boxoffice/views/admin.py

```diff
@@ -26,9 +27,12 @@ def index():
 
 def jsonify_org(data):
     item_collections_list = ItemCollection.query.filter(ItemCollection.organization == data['org']).order_by('created_at desc').all()
+    form = ItemCollectionForm()
```

Comment:
This variable isn't required since it's only being used once. Invoke `ItemCollectionForm()` directly on L31?

Reply:
Fixed